src/_xoConnection.js

```diff
@@ -131,6 +131,35 @@ class XoConnection extends Xo {
     return id;
   }
 
+  async runBackupJob(jobId, scheduleId, { remotes, nExecutions = 1 }) {
+    for (let i = 0; i < nExecutions; i++) {
+      await xo.call("backupNg.runJob", { id: jobId, schedule: scheduleId });
+    }
+    const backups = {};
+    if (remotes) {
+      const backupsByRemote = await xo.call("backupNg.listVmBackups", {
+        remotes,
+      });
+      for (const remote in backupsByRemote) {
```

Comment:
You can also use `forOwn`

Reply:
But in first, we just want the `id` of the remote.
Don't you think it is simpler to use `for` ? If you don't, I will change it with `forOwn`.

Reply:
Is just for being coherent. If we use `forOwn` for the bellow loops better use it also for this. `forOwn` also provide the key for each iteration.